### PR TITLE
chore: add html next.js headers

### PR DESCRIPTION
**Pull request by kushthedude**
Signed-off-by: Kush Trivedi <kushthedude@gmail.com>

### CODE REVIEW

File: provider-ui/pages/_document.js [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
+import Document, { Head, Main, NextScript, Html } from 'next/document';
 import flush from 'styled-jsx/server';
 
 class MesheryDocument extends Document {
   render() {
     // eslint-disable-next-line no-unused-vars
     const { pageContext } = this.props;
     return (
-      <html lang="en" dir="ltr">
+      <Html lang="en" dir="ltr">
```
Comment (leecalcote) [inline provider-ui/pages/_document.js:11]: Interesting. Is the Capitalization significant?

Comment (kushthedude) [reply provider-ui/pages/_document.js:11]: Html is the next.js document wrappers and html is what plainular html means. Html has some added meta tags enhanced for the next

Comment (leecalcote) [reply provider-ui/pages/_document.js:11]: Oh, I see. Hmpf.

**Review by leecalcote — APPROVED** [review]
LGTM

**Pull request merged**